totalRP3/Core/Configuration.lua

```diff
@@ -378,12 +378,17 @@ TRP3_API.events.listenToEvent(TRP3_API.events.WORKFLOW_ON_LOAD, function()
 		tinsert(localeTab, { locale:GetName(), locale:GetCode() });
 	end
 
+	--date format options
+	local date_format_settings_key = AddOn_TotalRP3.Enums.GENERAL_SETTINGS_CONFIG_KEYS.DATE_FORMAT;
+	local date_option_enum = AddOn_TotalRP3.Enums.DATE_OPTIONS;
+
 	registerConfigKey("heavy_profile_alert", true);
 	registerConfigKey("new_version_alert", true);
 	registerConfigKey("ui_sounds", true);
 	registerConfigKey("ui_animations", true);
 	registerConfigKey("default_color_picker", false);
 	registerConfigKey("increase_color_contrast", false);
+	registerConfigKey(date_format_settings_key, date_option_enum.D_M_Y);
```

Comment:
I'll need to think some more on the default here. That currently we're using D/M/Y makes this at least consistent, but I feel as if this should actually be "intelligent" and use a proper default based off client locale - but that might confuse US players if we now swapped D/M/Y to M/D/Y for them.

Reply:
I agree that making it smart would only confuse players on why their settings has changed without their approval